api/grpcserver/v2alpha1/interface.go

```diff
@@ -13,6 +14,7 @@ type malfeasanceInfo interface {
 
 type identityState interface {
 	All() map[types.NodeID][]identity.StateInfo
+	AllByOps(ops builder.Operations) map[types.NodeID][]identity.StateInfo
```

Comment:
You can probably remove All() and rename `AllByOps` to `All`, wdyt?
```suggestion
	All(ops builder.Operations) map[types.NodeID][]identity.StateInfo
```